src/profiler/format.ts

```diff
@@ -1,5 +1,5 @@
 import prettyTime from "pretty-time";
-import chalk from "chalk";
+import ansis from "ansis";
```

Comment:
We might use `getColor` from `consola/utils` which is already a dep

Reply:
Yeah! Lemme test that!

Reply:
@pi0 I've migrated most `ansis` w/ `consola/utils`: [`638c725` (#145)](https://github.com/unjs/webpackbar/pull/145/commits/638c725312efed8531b3b22ce4878ca0f6380cb6)

However, `src/util/cli.ts` use the hex color which `consola/utils` doesn't have support.

Reply:
@pi0 Will you add hex color support to `consolas/utils`? Otherwise we can't drop `ansis`.

Reply:
@pi0 Will it be a breaking change though?